compiler/bcq-tools/generate_bcq_output_arrays

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+
+import tensorflow as tf
+
+import argparse
+import sys
+
+
+def _get_parser():
+    """
+    Returns an ArgumentParser for generating output_arrays.
+    """
+    parser = argparse.ArgumentParser(
+        description=("Command line tool to generated output_arrays of BCQ nodes"))
+
+    # Input and output path.
+    parser.add_argument(
+        "-i",
+        "--input_path",
+        type=str,
+        help="Full filepath of the input file.",
+        required=True)
+    parser.add_argument(
+        "-o",
+        "--output_path",
+        type=str,
+        help="Full filepath of the output file.",
+        required=True)
+
+    return parser
+
+
+def load_graph(frozen_graph_filename):
+    """
+    Load graph from frozen pb file
+    """
+    with tf.compat.v1.gfile.GFile(frozen_graph_filename, "rb") as f:
+        graph_def = tf.compat.v1.GraphDef()
+        graph_def.ParseFromString(f.read())
+    with tf.Graph().as_default() as graph:
+        tf.import_graph_def(graph_def, name='')
+    return graph
+
+
+def dtype2str(dtype):
+    if dtype == "int32":
+        return "TF_INT32"
+    elif dtype == "int64":
+        return "TF_INT64"
+    elif dtype == "float32":
+        return "TF_FLOAT"
+    elif dtype == "bool":
+        return "TF_BOOL"
+    else:
+        raise Exception("Not supported dtype")
+
+
+def print_output_arrays(flags):
+    graph_model = load_graph(flags.input_path)
+    graph_model_def = graph_model.as_graph_def()
+    ops = graph_model.get_operations()
+
+    output_names = [op.outputs[0].name for op in ops 
+        if op.type == "Const" and "bcqinfo_" in op.outputs[0].name]
+
+    output_arrays = ""    
+    for output_name in output_names:
+        output_arrays += ","
+
+        colon_index = output_name.find(":")
+        if colon_index == -1:
+            output_arrays += output_name
+        else:
+            output_arrays += output_name[:colon_index]
```

Comment:
@llFreetimell What I'm gonna say is for general case. I think above code is for eliminating colon from its name. e.g. `input:0` -> `input`. But, AFAIK, there is no need to do this. Or maybe there's an another purpose of this.

Reply:
If I run following scripts, error occurs :(
```
$ python tf2tfliteV2.py --v1 --input_path /input/path --output_path /outout/path --input_arrays inputs_0 --output_arrays outputs_0:0

Traceback (most recent call last):
  File "/home/seok/ONE/draft_test/bin/tf2tfliteV2.py", line 200, in <module>
    main()
  File "/home/seok/ONE/draft_test/bin/tf2tfliteV2.py", line 196, in main
    _convert(flags[0])
  File "/home/seok/ONE/draft_test/bin/tf2tfliteV2.py", line 177, in _convert
    _v1_convert(flags)
  File "/home/seok/ONE/draft_test/bin/tf2tfliteV2.py", line 135, in _v1_convert
    _parse_array(flags.output_arrays), input_shapes)
  File "/home/seok/.local/lib/python3.5/site-packages/tensorflow/lite/python/lite.py", line 287, in from_frozen_graph
    sess.graph, output_arrays)
  File "/home/seok/.local/lib/python3.5/site-packages/tensorflow/lite/python/convert_saved_model.py", line 189, in get_tensors_from_tensor_names
    ",".join(invalid_tensors)))
ValueError: Invalid tensors 'outputs_0:0' were found.
```

Reply:
@llFreetimell Oh, I got a mistake. Actually, with `--v2` option, it works well like what I said, but `--v1` doesn't.